src/main/java/mate/academy/springboot/swagger/controller/ProductController.java

```diff
@@ -30,64 +32,75 @@
 public class ProductController {
     private final ProductService productService;
     private final ProductMapper productMapper;
+    private final OrderParser orderParser;
 
     @PostMapping
     @ApiOperation(value = "create a new product")
-    ProductResponseDto create(@RequestBody ProductRequestDto reqDto) {
-        return productMapper.toRespDto(productService.save(productMapper.toModel(reqDto)));
+    public ProductResponseDto create(
+            @RequestBody(description = "product to create", required = true,
+                    content = @Content(schema = @Schema(implementation = ProductRequestDto.class)))
+            ProductRequestDto requestDto
+    ) {
+        Product product = productService.save(productMapper.toModel(requestDto));
+        return productMapper.toRespDto(product);
     }
 
     @GetMapping("/{id}")
     @ApiOperation(value = "find product by id")
-    ProductResponseDto findById(@PathVariable Long id) {
+    public ProductResponseDto findById(@PathVariable Long id) {
         return productMapper.toRespDto(productService.findById(id));
     }
 
     @DeleteMapping("/{id}")
     @ApiOperation(value = "delete product by id")
-    boolean deleteById(@PathVariable Long id) {
+    public boolean deleteById(@PathVariable Long id) {
         return productService.deleteById(id);
     }
 
     @PutMapping("/{id}")
     @ApiOperation(value = "update product by id")
-    ProductResponseDto updateById(@PathVariable Long id, @RequestBody ProductRequestDto reqDto) {
-        Product product = productMapper.toModel(reqDto);
+    public ProductResponseDto updateById(
+            @PathVariable Long id,
+            @RequestBody(description = "product to create", required = true,
```

Comment:
copy pasta is evil, let's try to avoid it :)
```suggestion
            @RequestBody(description = "product to update", required = true,
```